pages/fiscal-sponsorship/climate/index.js

```diff
@@ -1367,15 +1286,15 @@ export class Organization {
    * @type {boolean}
    */
   get is128Recommended() {
-    return this.raw.partners?.['128_collective']?.recommended
+    return false
   }
 
   /**
    * Checks if the organization is funded by 128 Collective.
    * @type {boolean}
    */
   get is128Funded() {
-    return this.raw.partners?.['128_collective']?.funded
+    return false
   }
```

Comment:
Let's see if we can remove these methods all together. Separate PR is fine.

Same things with the similar methods in `pages/fiscal-sponsorship/directory/index.js`

Reply:
doing a quick search, i don't think these methods were ever used